src/storage/storage.ts

```diff
@@ -734,7 +734,7 @@ export class Storage implements IStorage {
     }
     this.version = null;
     if (cleanStorage || cleanAddresses) {
-      this.cleanStorage(cleanStorage, cleanAddresses);
+      await this.cleanStorage(cleanStorage, cleanAddresses);
```

Comment:
The `cleanStorage` method calls the first parameter `cleanHistory`, which I think is more accurate than `cleanStorage`.

What do you think about renaming this here as well? Would it be within the scope of this PR?

Reply:
We should make it outside this PR, but I agree this is a better name